1.12/Dockerfile

```
@@ -0,0 +1,28 @@
+# vim:set ft=dockerfile:
+
+FROM cimg/base:2021.04
+
+LABEL maintainer="Community & Partner Engineering Team <community-partner@circleci.com>"
+
+# Install Erlang via Erlang Solutions' .deb
+ENV ERLANG_VERSION="23.1"
```

Can this use 24.0 to match this official Elixir images?

```suggestion
ENV ERLANG_VERSION="24.0"
```